docroot/modules/humsci/hs_paragraph_types/hs_paragraph_types.module

```diff
@@ -31,3 +33,30 @@ function hs_paragraph_types_preprocess_field__field_hs_collection_items(&$variab
   $variables['attributes']['class'][] = "item-per-row--$items_per_row";
   $variables['#attached']['library'][] = 'hs_paragraph_types/hs_collection';
 }
+
+/**
+ * Implements hook_field_widget_single_element_WIDGET_TYPE_form_alter().
+ */
+function hs_paragraph_types_field_widget_single_element_paragraphs_form_alter(&$element, &$form_state, $context) {
```

Comment:
`$element` and `$context` should be typehinted.
`$form_state` is a FormStateInterface. It should be typehinted and doesn't need to be passed by reference.

Reply:
Moved to `hs_paragraph_types_field_widget_complete_paragraphs_form_alter` and fixed there